app/src/organisms/ProtocolDetails/index.tsx

```diff
@@ -344,6 +345,17 @@ export function ProtocolDetails(
     setShowChooseRobotToRunProtocolSlideout(true)
   }
 
+  const UNKNOWN_ATTACHMENT_ERROR = `${protocolDisplayName} protocol uses 
+  instruments  or modules from a future version of Opentrons software. Please update 
```

Comment:
The space between `instruments` and `or` is 2-space. Is this on purpose?
If not, ↓

```suggestion
  instruments or modules from a future version of Opentrons software. Please update 
```